src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -93,11 +93,17 @@ public function theStoreHasCurrency($currencyCode)
 
     /**
      * @Given the store has currency :currencyCode, :secondCurrencyCode
+     * @Given the store has currency :currencyCode and :secondCurrencyCode
+     * @Given the store has currency :currencyCode, :secondCurrencyCode and :thirdCurrencyCode
      */
-    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode)
+    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode, $thirdCurrencyCode = null)
```

Comment:
``` php
/**
 * @Given the store has currency :currencyCode, :secondCurrencyCode
 * @Given the store has currency :currencyCode and :secondCurrencyCode
 * @Given the store has currency :currencyCode, :secondCurrencyCode and :thirdCurrencyCode
 */
public function theStoreHasCurrencyAnd(...$currencies)
{
    foreach ($currencies as $currency) {
        $this->saveCurrency($this->createCurrency($currency));
    }
}
```

![image](https://cloud.githubusercontent.com/assets/9448101/20544831/2fd41e06-b10c-11e6-84e2-1ea6117e8547.png)

Seems like that's a no.
Cleared all caches and other stuff that might cause some problems, but with no gain.
If you'd be kind enough to test it at your end it'd be great 🍻